utils/generate_html_pages.sh

```diff
@@ -60,9 +62,10 @@ fi
 pushd build/tables
 touch index.html
 echo "<html>" > index.html
```

Comment:
```suggestion
echo "<!DOCTYPE html>" > index.html
echo '<html lang="en">' >> index.html
echo '<meta charset="utf-8" />' >>
```
If we want this to be a valid HTML5 document.